sympy/series/limits.py

```diff
@@ -212,36 +211,49 @@ def doit(self, **hints):
         if not e.has(z):
             return e
 
+        if e.is_meromorphic(z, z0):
+            if abs(z0) is S.Infinity:
+                newe = e.subs(z, -1/z)
+            else:
+                newe = e.subs(z, z + z0)
+            try:
+                coeff, exp = newe.leadterm(z)
+                if coeff is not S.ComplexInfinity:
+                    if exp > 0:
+                        return S.Zero
+                    elif exp == 0:
+                        return coeff
+                    if str(dir) == "+" or not(int(exp) & 1):
+                        return S.Infinity*sign(coeff)
+                    elif str(dir) == "-":
+                        return S.NegativeInfinity*sign(coeff)
```

Comment:
There is a third alternative "+-". At this point `exp` is odd so the limit should be `zoo`.

Reply:
So something like:
```
else:
    return S.ComplexInfinity
```

Reply:
Yes. It seems that `Limit` already checks that there are no other alternatives.